### DISCUSSION TITLE
H265 stream doesn't play in NVR

### DISCUSSION THREAD

**Discussion by bluesky2319** [bluenviron/mediamtx#1125] [Questions]
Hello, 

I use RTSP simple server as my RTSP server for restreaming the transcoded stream from the IP camera using FFmpeg. In my project after receiving the stream from IP cameras, I do transcoding for decreasing its bandwidth, and after transcoding I restream it. 

Everything is fine when I use players like VLC for playing the transcoded stream (h264 and h265). But in the NVR there is a problem, only I can play the h264 (transcoded) stream, and I can't play the h265 transcoded stream. I noticed NVR (Hikvision) knows my h265 stream as an h264 stream! 

The NVR can play the original h265 stream from an IP camera without using RTSP simple server. I tried with some cameras (different brands)  and the NVR is OK with their h265 original streams.

I checked it with Dahua NVR, and the same issue is in there plus the h264 stream can't play smoothly, it has some lags, and the framerate is 1 fps!

My FFmpeg version is 5.1 and I see this error in the FFmpeg:
`[rtsp @ 000001febc5f9500] method SETUP failed: 461 Unsupported Transport`

The FFmpeg command is:
`ffmpeg -rtsp_transport tcp -i "rtsp://admin:Password1@192.168.1:12:554/Steaming1/Channel1/101" -vcodec hevc -profile:v main -b:v 500k -preset veryfast -acodec aac -b:a 32k -f rtsp "rtsp://localhost:8554/high"`

I would appreciate any help to solve the issue, playing the transcoded stream by RTSP Simple server/FFmpeg.

**Comment by aler9** [comment]
Hello, please open an issue and provide relevant data (logs between the server and the NVR) in order to allow us to identify the problem, thanks.
